### PR TITLE
36638 - Create book

### PR DESCRIPTION
**Pull request by luongvietdung**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,8 +26,15 @@ gem "jbuilder", "~> 2.0"
 gem "sdoc", "~> 0.4.0", group: :doc
 
 gem "devise"
+
```
Comment (ThuBM) [inline Gemfile]: remove

File: Gemfile [hunk 2]
```diff
@@ -26,8 +26,15 @@ gem "jbuilder", "~> 2.0"
...
 gem "will_paginate"
```
Comment (ThuBM) [inline Gemfile]: remove

File: Gemfile [hunk 3]
```diff
@@ -26,8 +26,15 @@ gem "jbuilder", "~> 2.0"
...
 
+gem "jquery-ui-rails"
+
```
Comment (ThuBM) [inline Gemfile]: remove

File: Gemfile [hunk 4]
```diff
@@ -26,8 +26,15 @@ gem "jbuilder", "~> 2.0"
...
+gem "bootsy"
+
```
Comment (ThuBM) [inline Gemfile]: remove

File: Gemfile [hunk 5]
```diff
@@ -26,8 +26,15 @@ gem "jbuilder", "~> 2.0"
...
+gem "mini_magick"
+
```
Comment (ThuBM) [inline Gemfile]: remove

File: app/models/book.rb [hunk 6]
```diff
@@ -1,11 +1,15 @@
 class Book < ActiveRecord::Base
+  include Bootsy::Container
   belongs_to :category
```
Comment (ThuBM) [inline app/models/book.rb]: add new line

File: app/views/admin/books/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,39 @@
+<div class="row">
+  <div class="col-md-12 block">
+    <h1><%= t ".create_book_title" %></h1>
+    <%= form_for [:admin, @book], html: {multipart: true} do |f| %>
+      <%= render "shared/form_errors", object: f.object %>
```
Comment (ThuBM) [inline app/views/admin/books/new.html.erb]: @book

File: app/models/book.rb [hunk 8]
```diff
@@ -1,11 +1,16 @@
...
   has_many :user_books, dependent: :destroy
 
+  mount_uploader :image, CoverUploader
+
   validates :title, presence: true
   validates :published_date, presence: true
   validates :author, presence: true
   validates :description, presence: true
   validates :number_of_page, presence: true, numericality: {minimum: 1}
+  validates :category_id, presence: true
```
Comment (trinhducduy) [inline app/models/book.rb]: validates :category instead

**Comment by ThuBM** [comment]
ok

**Pull request merged**